jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -137,6 +139,9 @@ public Response crossOriginForInstallations(@Context HttpHeaders headers) {
      * @param entity    {@link Installation} for Device registration
      * @return          registered {@link Installation}
      *
+     * @requestheader x-ag-old-token the old push service dependant token (ie InstanceID in FCM).  
```

Comment:
unfortunaltey, we also need a JavaDoc `@param` here, since the build is failing :/(